src/routes/(pages)/app/(app)/components/filecoin-explore-page.svelte

```diff
@@ -1,3 +1,8 @@
+<script lang="ts" context="module">
+  // DEFAULT_EXPLORE_PAGE_FEATURED_PROJECT_FRAGMENT
+  // shared with load-default-explore-page
+</script>
```

Comment:
I think _ideally_ there would be a component for `ExplorePageFeaturedProjectSection` that would export this fragment, and that would simply be imported both here and on the default explore page. But I'm also fine with this for now 👍

Reply:
Dang, of course that makes total sense. I'll do a fast follow